### ISSUE TITLE
Any guide for c630?

### ISSUE THREAD

**Issue by falkor2k15** [aarch64-laptops/build#36]
I booted into the Ubuntu 18.04 installer in trial mode - but neither the desktop installer nor gparted can see the eUFS disk. I have shrunken the windows partition from within Windows - but what's the easiest way can I get Ubuntu onto the unallocated space and ensure GRUB will work when I change the boot manager?

**Comment by lag-linaro** [comment]
Have you read the README for this repo?

https://github.com/aarch64-laptops/build#installing

**Comment by falkor2k15** [comment]
Yeah I've read it over and over, but it doesn't seem to apply to the c630 or several steps have been omitted, hence I opened this issue.

**Comment by lag-linaro** [comment]
Please provide some more information. For instance:

- In what way do you think it does not apply?
- Which steps have been omitted?
- What parts are you finding confusing?

**Comment by schmidtbag** [comment]
I realize this is a issue is a bit old, but I think what falkor2k15 was really getting at here is that the documentation seems rather outdated, and all of the valuable+current information is scattered all over the place (much of which I find rather hard to find).  It's also tedious to sift through which of that decentralized documentation is actually relevant today.  There are people who made progress with things like wifi, audio, and accelerated graphics, even though the main /build page says otherwise.
I'd like to help with this project but it's such a chore trying to play catch-up with everyone else, particularly for things that have already been known to work.  That being said, it isn't even clear to me what specifically isn't actually working yet.

But... updated documentation might not be that necessary, if instead there was a disk image for Ubuntu 20.04, 5.7 or 5.8 kernel, with all known-working patches.  The most recent image was posted in September 2019, and a lot of stuff relevant to these laptops has been put into the mainline kernel since then.
If community members can be provided a more up-to-date and patched disk image (preferably, without a GUI installed), I feel it'll be much faster and easier for others to join in.

**Comment by lag-linaro** [comment]
Sounds good.  Any contributions will be gratefully received.
